session4/eevee.py

```diff
@@ -0,0 +1,9 @@
+class Eevee:
+    def __init__(self,name,type): 
```

Comment:
Nitpick for consistency: add spaces between commas
```suggestion
    def __init__(self, name, type): 
```